pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -1055,6 +1055,25 @@ typedef struct pjsua_call_setting
      */
     unsigned         vid_cnt;
 
+    /**
+     * Media direction. This setting will only be used if the flag
+     * PJSUA_CALL_SET_MEDIA_DIR is set, and it will persist for subsequent
+     * offers or answers.
```

Comment:
- "it will persist for subsequent offers or answers" may cause misinterpretation that the media dir setting cannot be updated later?
- To make directions persistence, shouldn't `PJSUA_CALL_SET_MEDIA_DIR` be reset in `pjsua_call_cleanup_flag()`?